html/pfappserver/lib/pfappserver/PacketFence/Controller/Config/Fingerbank/DB.pm

```diff
@@ -30,12 +30,10 @@ sub update :Local :Args(0) :AdminRole('FINGERBANK_UPDATE') {
 
     $c->stash->{current_view} = 'JSON';
 
-    my ( $status, $status_msg ) = fingerbank::DB::update_upstream;
-
-    pf::fingerbank::sync_upstream_db();
+    my $apiclient = pf::api::jsonrpcclient->new;
+    $apiclient->notify('fingerbank_update_upstream_db');
 
-    $c->stash->{status_msg} = $status_msg;
-    $c->response->status($status);
+    $c->stash->{status_msg} = $c->loc("Fingerbank upstream DB update request successfully dispatched");
```

Comment:
I'd say a report e-mail will be sent ? Or it's over-detailing ?

Reason why I would say it is if it's the first time I run it and I don't know an e-mail is sent, the absence of e-mail is not alarming to me